packages/sources/src/addSourceWizard/FinalWizard.js

```diff
@@ -93,6 +93,12 @@ const FinalWizard = ({
         step = <ErroredStep
             onClose={ afterError }
             primaryAction={tryAgain}
+            secondaryActions={
+                <Text component='a' target="_blank" href="https://access.redhat.com/support/cases/#/case/new/open-case?caseCreate=true" rel="noopener noreferrer">
+                    {intl.formatMessage({ id: 'wizard.openTicket', defaultMessage: 'Open a support case' })}
+                </Text>
+            }
+            returnButtonTitle={ <FormattedMessage id="wizard.retryText" defaultMessage="Retry"/> }
```

Comment:
can you use `formatMessage` here as well? (just to have both messages using the same API)

Reply:
I plan to make a PR that will change this everywhere in wizard package after I done with these changes.

Reply:
alright then